bibliopixel/remote/server.py

```diff
@@ -5,36 +5,19 @@
 import os
 import logging
 from .. import log
+import sys
 
-# I want to keep this to supress the "Running on" message
-# But then it also swallows 404 erros and such in the console :(
-# werk_log = logging.getLogger('werkzeug')
-# werk_log.setLevel(logging.ERROR)
+if '--verbose' not in sys.argv:
```

Comment:
Brutal, but sorta works.  :-D  (Now we have knowledge of what `--verbose` means scattered all over the code.)

We should avoid having anyone other than main/main.py touch `sys.argv`, but the contortions you'd need to do that would be extreme.  So don't change it.  :-D